index.html

```diff
@@ -1,12 +1,476 @@
+                            </div>
+                        </div>
+                        <div class="column">
+                            <div class="column-img">
+                                <img class="image" src="img/illustration-lack-of-direction.svg" />
+                            </div>
+                            <div class="column-subtitle">
+                                <h3>Lack of Action</h3>
+                            </div>
+                            <div class="column-desc">
+                                <p>Charting a career path can feel challenging at best. The process
+                                can feel hard and complex, causing employees to stop before they even start. Comfort zones
+                                are hard to break without the right guidance and support.</p>
+                            </div>
+                        </div>
+                        <div class="column">
+                            <div class="column-img">
+                                <img class="image" src="img/illustration-lack-of-support.svg" />
+                            </div>
+                            <div class="column-subtitle">
+                                <h3>Inefficient Support</h3>
+                            </div>
+                            <div class="column-desc">
+                                <p>Even when employees are presented with the right opportunity,
+                            career mobility is halted when they don’t have the right managerial support. Worse, they
+                            don’t have the tools or right connections to support their own career growth.</p>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        
+        <section class="one-column ">
+            <img class="lines" src="img/bg-lines.svg" />
+
+            <div class="container">
+                <div class="content">
+
+                    <div class="flex-column-align-center column">
+
+                        <div class="row flex column-subtitle">
+                            <p>Industry Report</p>
+                        </div>
+                        <div class="row column-title">
+                            <h3>To learn how Avenue helps your team overcome these challenges, checkout our industry
+                                report.</h3>
+                        </div>
+    
+                        <div class="row column-btn">
+                            <button class="button-download-report"><a class="text-button-download-report">Download Industry
+                                    Report</a></button>
+                        </div>
+                    </div>
+                </div>
+               
+
+            </div>
+        </section>
+
+        <section class="two-cards section-margin">
```

Comment:
Check designs one more time. Whole cards component has little bit darken background. Each card has a white background and border-radius. Check it out. To make this background sticky to previous component - remove section-margin from prev component, add paddings for cards compoent.